README.rst

```diff
@@ -40,46 +40,45 @@
 .. |NUMFocus| image:: https://img.shields.io/badge/powered%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A
 .. _NUMFocus: http://www.numfocus.org
 
+.. image:: doc/_static/logo2.png
 
-##########
-Matplotlib
-##########
+Matplotlib is a comprehensive library for creating static, animated, and interactive visualizations in Python. 
 
-Matplotlib is a Python 2D plotting library which produces publication-quality
-figures in a variety of hardcopy formats and interactive environments across
-platforms. Matplotlib can be used in Python scripts, the Python and IPython
-shell (à la MATLAB or Mathematica), web application servers, and various
-graphical user interface toolkits.
+Check out our `home page <http://matplotlib.org/>`_ for more information. 
+
+.. image:: doc/_static/readme_preview.png
 
-NOTE: The current master branch is now Python 3 only.  Python 2 support is
-being dropped.
+Matplotlib produces publication-quality figures in a variety of hardcopy formats
+and interactive environments across platforms. Matplotlib can be used in Python scripts, 
+the Python and IPython shell, web application servers, and various
+graphical user interface toolkits.
 
-`Home page <http://matplotlib.org/>`_
+**Note:** The master branch is Python 3 only.  Python 2 support has been dropped.
 
 Install
 =======
 
-For installation instructions and requirements, see the INSTALL.rst file or the
-`install <http://matplotlib.org/users/installing.html>`_ documentation. If you
-think you may want to contribute to matplotlib, check out the `guide to
-working with the source code
-<http://matplotlib.org/devel/gitwash/index.html>`_.
+For installation instructions and requirements, see `INSTALL.rst <INSTALL.rst>`_  or the
+`install <http://matplotlib.org/users/installing.html>`_ documentation. 
 
 Test
 ====
 
-After installation, you can launch the test suite::
+After installation, launch the test suite::
 
-  pytest
+  python -mpytest
```

Comment:
```suggestion
  python -m pytest
```

While technically correct, not using a space may confuse people less familiar with bash. We‘ve changed these also some time ago in the docs.